diff -urN Msql-Mysql-modules-1.2219.org/dbd/myMsql.h Msql-Mysql-modules-1.2219/dbd/myMsql.h
--- Msql-Mysql-modules-1.2219.org/dbd/myMsql.h	2001-10-31 04:35:03.000000000 +0100
+++ Msql-Mysql-modules-1.2219/dbd/myMsql.h	2005-01-08 00:10:22.694934550 +0100
@@ -117,7 +117,7 @@
 #define MyGetHostInfo(s) mysql_get_host_info(s)
 #define MyGetServerInfo(s) mysql_get_server_info(s)
 #define MyGetProtoInfo(s) mysql_get_proto_info(s)
-#define MyShutdown(s) mysql_shutdown(s)
+#define MyShutdown(s) mysql_shutdown(s, SHUTDOWN_DEFAULT)
 #ifdef REFRESH_GRANT
 #define MyReload(s) mysql_refresh(s, \
 				  REFRESH_GRANT|REFRESH_LOG|REFRESH_TABLES)
